client/android-java/app/src/main/res/xml/network_security_config.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<network-security-config>
```

Comment:
is this for Charles?

Reply:
IDK, but no need to make this work with Charles.  We do want this to work with a server running locally.